docs/src/content/docs/zh-cn/guides/sidebar.mdx

```diff
@@ -521,6 +521,52 @@ starlight({
 在多语言网站中，`slug` 的值不会包含 URL 的语言部分。
 例如，如果页面位于 `en/intro` 和 `pt-br/intro`，则在配置侧边栏时，slug 为 `intro`。
 
+### 通过徽章实现国际化
+
+对于[徽章](#徽章)，`text` 属性可以是字符串，而对于多语言网站，文本属性则可以是具有每个不同区域设置值的对象。
+使用对象形式时，键必须是 [BCP-47](https://www.w3.org/International/questions/qa-choosing-language-tags) 标签（例如 `en`、`ar` 或 `zh-CN`）：
+
+```js {11-16}
+starlight({
+	sidebar: [
+		{
+			label: '星座',
+			translations: {
+				'pt-BR': 'Constelações',
+			},
+			items: [
+				{
+					slug: 'constellations/andromeda',
+					badge: {
+						text: {
+							en: 'New',
```

Comment:
```suggestion
							zh-CN: '新',
```
Thanks for HiDeoo's suggeestion in [Line 542](https://github.com/withastro/starlight/pull/2405/files#r1774895578) !